gui/qt/main_window.py

```diff
@@ -211,7 +211,10 @@ def add_optional_tab(tabs, tab, icon, description, name, default=False):
             self.new_fx_history_signal.connect(self.on_fx_history)
 
         # update fee slider in case we missed the callback
-        self.fee_slider.update()
+        if (self.config.get('customfee') is None):
```

Comment:
Get rid of these parentheses in the if conditions, this is pointless for single clauses in Python and simply looks like the writer is more familiar with other languages.

Reply:
meme:  https://i.imgflip.com/wjrk5.jpg